packages/contracts/test/plugins/governance/majority-voting/token/token-voting.ts

```diff
@@ -1871,7 +1860,7 @@ describe('TokenVoting', function () {
       });
 
       it('early support criterium is sharp by 1 vote', async () => {
-        advanceIntoVoteTime(startDate, endDate);
+        time.increaseTo(startDate);
```

Comment:
missing `await`

Reply:
Fixed in 599b498e9a1c5208d3ef0e8a485fa2bd7d562fde